kiali/kiali
# Configurable server timeout

### Describe the change
- Update server timeout to be able to be greater than 30 seconds (The default)

### Steps to test the PR

Simulate a slow service, setting in handlers/tracing in WorkloadTraces/ServiceTraces/AppTraces: 
`
time.Sleep(40 * time.Second)`

Deploy Kiali: 

`make clean build-ui build cluster-push operator-create kiali-create -e CLUSTER_TYPE=minikube OPERATOR_INSTALL_KIALI=true`

Setup `server.timeout` to 50. 

The request of the workload traces should be successful after 50 seconds: 

![image](https://github.com/kiali/kiali/assets/49480155/2df9a3f9-571e-429a-9df6-5090e879b4d8)


### Automation testing

There is coverage already. 

### Issue reference

Fixes #7388
